x/assets/keeper/keeper.go

```diff
@@ -10,15 +10,18 @@ import (
 type Keeper struct {
 	storeKey storetypes.StoreKey
 	cdc      codec.BinaryCodec
+	assetstype.OracleKeeper
 }
 
 func NewKeeper(
 	storeKey storetypes.StoreKey,
 	cdc codec.BinaryCodec,
+	oKeeper assetstype.OracleKeeper,
```

Comment:
let's use oraKeeper / oracleKeeper instead of oKeeper.

Reply:
updated